.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,13 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <JetCodeStyleSettings>
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+    </JetCodeStyleSettings>
+    <codeStyleSettings language="kotlin">
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+      <indentOptions>
+        <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      </indentOptions>
+    </codeStyleSettings>
```

Comment:
I "guessed" this from the other files. This puts indent as 4 spaces. I usually tend to like smaller indents better but 4 spaces work too :) 

![0_rWvhTR1T46HOPSzO](https://user-images.githubusercontent.com/3974977/188516037-d61e8d34-a8e1-4ee7-8cd0-424eea42f8ec.gif)